trunk/SUAVE/Methods/Flight_Dynamics/Static_Stability/compute_aero_derivatives.py

```diff
@@ -168,6 +168,81 @@ def compute_aero_derivatives(segment):
+                iterate.conditions(perturbed_segment)                 
+                
+                dDelta          = perturbed_segment.analyses.aerodynamics.geometry.wings[wing].control_surfaces[cs].deflection - segment.analyses.aerodynamics.geometry.wings[wing].control_surfaces[cs].deflection
+                dCL             = perturbed_segment.state.conditions.aerodynamics.lift_coefficient - segment.state.conditions.aerodynamics.lift_coefficient
+                dCD             = perturbed_segment.state.conditions.aerodynamics.drag_coefficient - segment.state.conditions.aerodynamics.drag_coefficient
+                
+                # roll and yaw moment coefficient derivatives
+                if surrogate_used:
+                    print("Surrogate model is being used. No roll or yaw coefficients available.")
+                    dCn = 0
+                    dCl = 0
+                else:
+                    # use VLM outputs directly
+                    dCn = perturbed_segment.state.conditions.stability.static.yawing_moment_coefficient - segment.state.conditions.stability.static.yawing_moment_coefficient
+                    dCl = perturbed_segment.state.conditions.stability.static.rolling_moment_coefficient - segment.state.conditions.stability.static.rolling_moment_coefficient              
+                
+                # propeller derivatives 
+                dCT, dCP = propeller_derivatives(segment, perturbed_segment, n_cpts) 
+                    
+                dCL_dDelta      = dCL/dDelta 
+                dCD_dDelta      = dCD/dDelta
+                dCn_dDelta      = dCn/dDelta            
+                dCl_dDelta      = dCl/dDelta           
+                dCT_dDelta      = dCT/dDelta        
+                dCP_dDelta      = dCP/dDelta
+                dCL_dDelta_tag  = 'dCL_dDelta_'+cs
+                dCD_dDelta_tag  = 'dCD_dDelta_'+cs
+                dCn_dDelta_tag  = 'dCn_dDelta_'+cs  
+                dCl_dDelta_tag  = 'dCl_dDelta_'+cs   
+                dCT_dDelta_tag  = 'dCT_dDelta_'+cs   
+                dCP_dDelta_tag  = 'dCP_dDelta_'+cs  
+                segment.state.conditions.aero_derivatives[dCL_dDelta_tag]  = dCL_dDelta
+                segment.state.conditions.aero_derivatives[dCD_dDelta_tag]  = dCD_dDelta
+                segment.state.conditions.aero_derivatives[dCn_dDelta_tag]  = dCn_dDelta
+                segment.state.conditions.aero_derivatives[dCl_dDelta_tag]  = dCl_dDelta
+                segment.state.conditions.aero_derivatives[dCT_dDelta_tag]  = dCT_dDelta
+                segment.state.conditions.aero_derivatives[dCP_dDelta_tag]  = dCP_dDelta
+
+    # ----------------------------------------------------------------------------    
+    # Velocity magnitude perturbation
+    vinf                = segment.state.conditions.frames.inertial.velocity_vector
+    vmag                = np.linalg.norm(vinf,axis=1)
+    psi                 = np.arctan2(vinf[:,2],vinf[:,0])
```

Comment:
I think this is gamma, the flight path angle. PSI is the heading angle

Reply:
Done, thanks!